packages/eslint-config/eslint-strapi.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  root: true,
+  extends: [
+    // Load common config
+    "./eslint-config.js",
+    // Load pagopa eslint config
+    "@pagopa/eslint-config/strong",
+  ],
+  overrides: [
+    {
+      files: ['**/*.ts'],
+      rules: {
+        'functional/no-method-signature': 'off',
+        'functional/immutable-data': 'off',
+        'functional/prefer-readonly-type': 'off',
+        'functional/no-expression-statements': 'off',
+        'functional/no-return-void': 'off',
+        'prefer-arrow/prefer-arrow-functions': 'off',
```

Comment:
**question**: why disable this rule?

Isn't there a way to convert our code using the arrow function?
In this way we are coherent with the rest of our codebase.

Reply:
Our code is basically taken from strapi docs, strapi has a more OO approach and ther's no way to functioanlly trhow an exception, or to do a side effect on a strapi object. I mean we could still do it, but that would make the code basically unreadable and way more verbose than it needs to be. 

Functional programming is amazing, 
Functional programming a OOP framework is a nightmare